libpysal/io/iohandlers/tests/test_db.py

```diff
@@ -4,43 +4,26 @@
 import unittest as ut
 from .... import examples as pysal_examples
 
+from shapely.geometry import Point
+from shapely import wkb
+
 try:
     import sqlalchemy
 
     missing_sql = False
 except ImportError:
     missing_sql = True
 
-try:
-    import geomet
-
-    missing_geomet = False
-except ImportError:
-    missing_geomet = True
-
-
-def to_wkb_point(c):
-    """Super quick hack that does not actually belong in here."""
-
-    point = {"type": "Point", "coordinates": [c[0], c[1]]}
-
-    return geomet.wkb.dumps(point)
-
 
-@ut.skipIf(
-    missing_sql or missing_geomet,
-    "Missing dependencies: Geomet ({}) & SQLAlchemy ({}).".format(
-        missing_geomet, missing_sql
-    ),
-)
+@ut.skipIf(missing_sql, f"Missing dependency: SQLAlchemy ({missing_sql}).")
 class Test_sqlite_reader(ut.TestCase):
     def setUp(self):
         path = pysal_examples.get_path("new_haven_merged.dbf")
         if path is None:
             pysal_examples.load_example("newHaven")
             path = pysal_examples.get_path("new_haven_merged.dbf")
         df = pdio.read_files(path)
-        df["GEOMETRY"] = df["geometry"].apply(to_wkb_point)
+        df["GEOMETRY"] = df["geometry"].apply(lambda p: wkb.dumps(Point(p)))
```

Comment:
```suggestion
        df["GEOMETRY"] = shapely.to_wkb(shapely.points(df["geometry"]))
```
or something like that. you can completely avoid apply loop

Reply:
So thing here is that `df["geometry"]` is not `Point` objects yet. Let me see about a workaround.

Reply:
I'm aware but I wasn't sure which object it is, that is why I said "something like that" :D.

Reply:
OK, needed some massaging. The following works:
```python
shapely.to_wkb(shapely.points(df["geometry"].values.tolist()))
```